language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -251,18 +251,40 @@ class DottyLanguageServer extends LanguageServer
     val enclTree = Interactive.enclosingTree(driver.openedTrees(uri), pos)
     val sym = Interactive.sourceSymbol(enclTree.symbol)
 
-    if (sym == NoSymbol) Nil.asJava
-    else {
-      val (trees, include) =
-        if (enclTree.isInstanceOf[MemberDef])
-          (driver.allTreesContaining(sym.name.sourceModuleName.toString),
-           Include.overriding | Include.overridden)
-        else
-          (SourceTree.fromSymbol(sym.topLevelClass.asClass).toList,
-           Include.overriding)
-      val defs = Interactive.namedTrees(trees, include, sym)
-      defs.map(d => location(d.namePos)).asJava
-    }
+    val defs =
+      if (sym == NoSymbol) Nil
+      else enclTree match {
+        case imp: Import =>
+          def lookup(name: Name): Symbol = {
+            imp.expr.tpe.member(name).symbol
+          }
+          val importedSyms = imp.selectors.flatMap {
```

Comment:
My guess would be that there is no more than one selector at the cursor position. So you should be able to do `imp.selectors.find(_.pos.contains(pos.pos))`.